### ISSUE TITLE
Delegating functionality to objcopy

### ISSUE THREAD

**Issue by deliciouslytyped** [rui314/mold#6]
https://github.com/rui314/mold/blame/830a9985a887e83ca5435cd69d3bfd9189f18960/README.md#L158 suggests most things should be doable with objcopy. From what I recall / have seen, objcopy doesn't deal with rewriting dynamic symbols at all.

Once again I am ignorant about internals and feasibility, but the main reason I went down the linker script rabbit hole is that I couldn't (and still can't) figure out how to hide a dynamic symbol. In my case I wanted to hide an undefined import that shouldn't have even existed. (largely as a debugging measure, given that this suggested problems in the upstream compilation.

**Comment by rui314** [comment]
You can use version script to hide or export symbols. Specifically,

```{ local: *; global: foo; bar; baz; }```

exports only foo, bar and baz from your shared object file or executable and hides everything else.

**Comment by deliciouslytyped** [comment]
Ok thanks, I'll have to try that at some point. I saw hints about this but nothing I managed to make much sense of.

**Comment by rui314** [comment]
Yeah, you should be able to flip the condition, like this `{ global: *; local: foo; bar; baz; }`.
